<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>887</width>
    <height>737</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QLabel" name="lFunc">
      <property name="text">
       <string>Test Function</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelSrcFile">
      <property name="text">
       <string>Source File</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="comboFunc"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="editTestFile">
      <property name="text">
       <string>/Users/liuzikai/Documents/Courses/AST/AST-Project/ast-klee/examples/klee-unit/get_sign_test.c</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="btnAnalyzeSrc">
      <property name="text">
       <string>Analysis File</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QGroupBox" name="groupRet">
      <property name="title">
       <string>Return Value</string>
      </property>
      <layout class="QVBoxLayout" name="verticalRet"/>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QGroupBox" name="groupArgs">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Arguments</string>
      </property>
      <layout class="QGridLayout" name="gridArgs"/>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="btnGenerateDriver">
      <property name="text">
       <string>Generate Test Driver</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="btnAnalyzeFunc">
      <property name="text">
       <string>Analysis Function</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="editSrcFile">
      <property name="text">
       <string>/Users/liuzikai/Documents/Courses/AST/AST-Project/ast-klee/examples/klee-unit/get_sign.c</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="labelTestFile">
      <property name="text">
       <string>Working Test File</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="btnGenerateKLEE">
      <property name="text">
       <string>Generate Test Cases</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0" colspan="3">
     <widget class="QGroupBox" name="groupTestCases">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Test Cases</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QLabel" name="labelTestName">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="btnAccept">
         <property name="text">
          <string>Accept Selected</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="editTestName"/>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QTableWidget" name="tableTests"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>887</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
